source/_integrations/template.markdown

```diff
@@ -68,9 +68,6 @@ template:
 
 {% endraw %}
 
-### Template and action variables
-
-State-based template entities have the special template variable `this` available in their templates and actions. The `this` variable aids [self-referencing](#self-referencing) of an entity's state and attribute in templates and actions.
```

Comment:
We should not remove this section because it's a subsection of "State-based template binary sensors, buttons, numbers, selects and sensors"